### ISSUE TITLE
Failing unstable build

### ISSUE THREAD

**Issue by kaustubh-nair** [publiclab/mapknitter#952]
The build has been failing for quite some time on mapknitter unstable with this error
```

ERROR: for web  Cannot create container for service web: Conflict. The container name "/mapknitter" is already in use by container "560274c0e70a735026b595c48ab3571c00736456734b8676a8fda6a0a7bc182a". You have to remove (or rename) that container to be able to reuse that name.
Encountered errors while bringing up the project.
Makefile:22: recipe for target 'redeploy-container' failed
make: *** [redeploy-container] Error 1
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```
Here's a link to a failing build https://jenkins.laboratoriopublico.org/job/Mapknitter-Unstable/200/console

**Comment by kaustubh-nair** [comment]
Pinging @icarito @alaxalves and @jywarren here..

**Comment by alaxalves** [comment]
I have no acces to the Jenkins machine :(. I can't help much with this. There is a dangling MK container in our Jenkins machine. If we get access to the machine the solution is just removing the dangling container and redeploy it, we can do this through the Makefile script, but wouldn't advise you to do that

**Comment by alaxalves** [comment]
I think @icarito is already into this.

**Comment by jywarren** [comment]
OK, http://mapknitter-unstable.laboratoriopublico.org/#6/20.120/-99.005/ is back online, though it's showing an error message. Can someone try pushing something locally confirmed to run here?

**Comment by jywarren** [comment]
Also linking to #953...

**Comment by icarito** [comment]
I think this has something to do with `container_name` appearing in `docker-compose.yml` file. We don't want to name containers because they will conflict with each other on the same host, this is what is stopping stable now (see #953).

**Comment by icarito** [comment]
I see @alaxalves why you added container_name, in order to be able to `wait_for_container` on the Makefile. But this breaks in our staging setup because there will be more than one Mapknitter web container.

**Comment by icarito** [comment]
Currently Unstable Makefile is stuck trying to start Mapknitter https://jenkins.laboratoriopublico.org/job/Mapknitter-Unstable/210/console because of this

**Comment by icarito** [comment]
This should be fixed with #983 
:balloon: :tada:
